services/terraformer/server/loadbalancer/loadbalancer.go

```diff
@@ -72,27 +72,27 @@ func (l LBcluster) Build() error {
 
 func (l LBcluster) Destroy() error {
 	cluster := clusterBuilder.ClusterBuilder{
-		//DesiredInfo: , //desired state is not used in DestroyNodepools
+		//DesiredInfo: desired state is not used in Destroy
 		CurrentInfo: l.CurrentLB.ClusterInfo,
 		ProjectName: l.ProjectName,
-		ClusterType: pb.ClusterType_LB}
-	nodeIPs := getNodeIPs(l.CurrentLB.ClusterInfo.NodePools)
+		ClusterType: pb.ClusterType_LB,
+	}
+	if err := cluster.DestroyNodepools(); err != nil {
```

Comment:
Maybe we can improve it a little by parallelizing it. `dns.DestroyDNSRecords()` doesn't need to wait until `cluster.DestroyNodepools()` returns.